api/common.go

```diff
@@ -17,7 +17,7 @@ import (
 // Common constants for daemon and client.
 const (
 	// Version of Current REST API
-	Version version.Version = "1.21"
+	Version version.Version = "1.20"
```

Comment:
my bad. putting it back. Needed to test with a newer server and the client doesn't like that.